frontend/src/services/logger.js

```diff
@@ -0,0 +1,13 @@
+import winston from "winston";
+
+const logger = new winston.Logger({
+  transports: [
+    new winston.transports.Console({
+      colorize: true,
+      level: process.env.LOG_LEVEL,
+      timestamp: Date.now(),
+    }),
+  ],
+});
+
+module.exports = logger;
```

Comment:
Does this methof of export work for you? This doesn't work for me. I'm thinking it's because `module.exports` is a `Node.js` thing and since we're in a React enviornment this can be changed to `export default logger`

Reply:
doh, changed in https://github.com/gambinish/stocks/pull/19/commits/4631c8f80e5ed8502207f2271da75c82ef69ea0e